account_loan_vendor_payment/models/account_payment.py

```diff
@@ -0,0 +1,26 @@
+from odoo import fields, models
+
+
+class AccountPayment(models.Model):
+    _inherit = "account.payment"
+
+    reconciled_bill_ids = fields.Many2many(store=True)
+    loans_count = fields.Integer(
+        compute="_compute_loans_count", string="Number of Loans"
+    )
+
+    loan_id = fields.Many2one("account.loan")
+
+    def _compute_loans_count(self):
+        self.loans_count = len(self.loan_id)
```

Comment:
**question**: why is this compute treated as if it's used by a recordset with a single item while the `_compute_get_loans_count` method it's implemented with more records in mind?